tests/ui/crashes/ice-5835.rs

```diff
@@ -0,0 +1,8 @@
+#![rustfmt::skip]
+
```

Comment:
```suggestion
#[rustfmt::skip]
```

To get rid of the rustc error.